src/SFA.DAS.Admin.Roatp.Web/Validators/Common/UkprnValidator.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FluentValidation;
+
+namespace SFA.DAS.Admin.Roatp.Web.Validators.Common;
+
+public static class UkprnValidator
+{
+    public const string UkprnFormatValidationMessage = "Enter a UKPRN using 8 digits";
+
```

Comment:
I'm concerned this message doesn't make clear this value must also start with 1. In the inner api, we use the message 'Ukprn must start with the value 1 and should be 8 digits long.', however this doesn't mean that's the correct message to use. Maybe ask BA to ask UX?